src/core/mount.c

```diff
@@ -53,6 +53,7 @@ static const UnitActiveState state_translation_table[_MOUNT_STATE_MAX] = {
 
 static int mount_dispatch_timer(sd_event_source *source, usec_t usec, void *userdata);
 static int mount_dispatch_io(sd_event_source *source, int fd, uint32_t revents, void *userdata);
+static int mount_dispatch_timeout_io(sd_event_source *source, usec_t usec, void *userdata);
```

Comment:
if it's a timeout , then it's not "io", hence shouldn't carry the name "io"...

I think a better name would be "mount_dispatch_proc_self_mountinfo_timer()" or so

Reply:
We are going to do some io (exactly the same way that mount_dispatch_io does) because there was a timeout... But I can also see that value of your suggestion - I'll go with that.